src/extension.ts

```diff
@@ -23,6 +23,7 @@ interface Config {
     username: string;
     login: string;
     headerType: string;
+    usePragmaOnce: Boolean;
```

Comment:
Just a little thing, "Boolean", if I'm correct, refers to the Javascript object and "boolean" refers to the Typescript type for it. So I might switch it to "boolean" as this is an interface declaration which is exclusive to Typescript.